ui/src/app/views/bug-report/component.ts

```diff
@@ -23,12 +28,16 @@ export class BugReportComponent implements OnInit {
     this.reporting = true;
     this.shortDescription = '';
     this.reproSteps = '';
+    this.profileService.getMe().subscribe(profile => {
+      this.contactEmail = profile.contactEmail;
```

Comment:
Just checking my interpretation: If the bug reporting tool is used when someone's not logged in (for example, early in the signup process), this will just be an empty field that someone can fill in?

Reply:
Correct.